src/test/scala/za/co/absa/hyperdrive/trigger/api/rest/WorkflowControllerTest.scala

```diff
@@ -31,10 +31,11 @@ import za.co.absa.hyperdrive.trigger.configuration.application.TestGeneralConfig
 import za.co.absa.hyperdrive.trigger.models.errors.ApiException
 import za.co.absa.hyperdrive.trigger.models.{Project, Workflow, WorkflowImportExportWrapper}
 
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.Future
+import scala.concurrent.{ExecutionContext, Future}
 
 class WorkflowControllerTest extends AsyncFlatSpec with Matchers with MockitoSugar with BeforeAndAfter {
+  override implicit def executionContext: ExecutionContext = scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
Probably we need to revisit the usage of the `AsyncFlatSpec` here. If we override its execution context, I guess we could just as well use `FlatSpec`. However, this is not a problem of upgrading to 2.12 and was like this before, so it's ok for me in this PR